### PR TITLE
Added german language + new contains rule

### PR DESCRIPTION
**Pull request by mintalicious**
After closing the last PR, I created a new one which contains the german language file, a new contains rule and little update of the required rule, which now checks the $value to not be empty instead of only not being null.

### CODE REVIEW

File: src/Validate.php [hunk 1]
```diff
@@ -547,6 +548,29 @@ public static function notIn(Validator $validator, $data, $pattern, $rule, $para
         }
     }
 
+    /**
+     * contains-only:<value>(,<value>)*
```
Comment (pdscopes) [inline src/Validate.php]: You forgot to update this part of the comment
```suggestion
     * contains:<value>(,<value>)*
```

File: src/Validate.php [hunk 2]
```diff
@@ -547,6 +548,29 @@ public static function notIn(Validator $validator, $data, $pattern, $rule, $para
...
+            if (null === $value || empty($value)) {
+                continue;
+            }
+            if (is_countable($value) && count($parameters) == count(array_intersect($value, $parameters))) {
+                continue;
+            }
+
+            $validator->addError($attribute, $rule, [':values' => implode(', ', $parameters)]);
+        }
+    }
```
Comment (pdscopes) [inline src/Validate.php:572]: Please could you add an integration test to cover this new validation?

See `tests/Integration/ValidateContainsOnlyTest.php`  as an example.

**Review by pdscopes — CHANGES_REQUESTED** [review]
Thanks for doing this so quickly, I have two comments for you to please address and then I will merge and release v3.

**Comment by mintalicious** [comment]
I fixed the PHPDoc for contains() and added an integration test file as suggested.
Please do not tag it as v3 - the PR for changing the return value of validate() is yes to come.

**Comment by mintalicious** [comment]
Btw: Thx for this nice little validation library ;-)

**Comment by pdscopes** [comment]
You are very welcome, I use it personally and I'm happy to share it!

**Review by pdscopes — APPROVED** [review]

**Comment by pdscopes** [comment]
Thanks again, this has now been release in [v2.7.0](https://github.com/pdscopes/php-form-validator/releases/tag/v2.7.0)

**Pull request merged**